modules/wim_features/wim_core/wim_core.strongarm.inc

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * @file
+ * wim_core.strongarm.inc
+ */
+
+/**
+ * Implements hook_strongarm().
+ */
+function wim_core_strongarm() {
+  $export = array();
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'admin_theme';
+  $strongarm->value = 'seven';
+  $export['admin_theme'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'theme_default';
```

Comment:
Let's set the default theme during profile install.

Municipality will then be able to override with their own feature. How about that?